syntax = "proto3";

message smallVarintDummy {
	int32	id1 	= 1;
	int32	id2 	= 2;
	int32	id3 	= 3;
	int32	id4 	= 4;
	int32	id5 	= 5;
	int32	id6 	= 6;
	int32	id7 	= 7;
	int32	id8 	= 8;
	int32	id9 	= 9;
	int32	id10 	= 10;
	int32	id11 = 11;
	int32	id12 = 12;
	int32	id13 = 13;
	int32	id14 = 14;
	int32	id15 = 15;
	int32	id16 = 16;
	int32	id17 = 17;
	int32	id18 = 18;
	int32	id19 = 19;
	int32	id20 = 20;
	int32	id21 = 21;
	int32	id22 = 22;
	int32	id23 = 23;
	int32	id24 = 24;
	int32	id25 = 25;
	int32	id26 = 26;
	int32	id27 = 27;
	int32	id28 = 28;
	int32	id29 = 29;
	int32	id30 = 30;
	int32	id31 = 31;
	int32	id32 = 32;
	int32	id33 = 33;
	int32	id34 = 34;
	int32	id35 = 35;
	int32	id36 = 36;
	int32	id37 = 37;
	int32	id38 = 38;
	int32	id39 = 39;
	int32	id40 = 40;
	int32	id41 = 41;
	int32	id42 = 42;
	int32	id43 = 43;
	int32	id44 = 44;
	int32	id45 = 45;
	int32	id46 = 46;
	int32	id47 = 47;
	int32	id48 = 48;
	int32	id49 = 49;
	int32	id50 = 50;
}

message biggerVarintDummy {
	int32	id1 = 1;
	int32	id2 = 2;
	int32	id3 = 3;
	int32	id4 = 4;
	int32	id5 = 5;
	int32	id6 = 6;
	int32	id7 = 7;
	int32	id8 = 8;
	int32	id9 = 9;
	int32	id10 = 10;
	int32	id11 = 11;
	int32	id12 = 12;
	int32	id13 = 13;
	int32	id14 = 14;
	int32	id15 = 15;
	int32	id16 = 16;
	int32	id17 = 17;
	int32	id18 = 18;
	int32	id19 = 19;
	int32	id20 = 20;
	int32	id21 = 21;
	int32	id22 = 22;
	int32	id23 = 23;
	int32	id24 = 24;
	int32	id25 = 25;
	int32	id26 = 26;
	int32	id27 = 27;
	int32	id28 = 28;
	int32	id29 = 29;
	int32	id30 = 30;
	int32	id31 = 31;
	int32	id32 = 32;
	int32	id33 = 33;
	int32	id34 = 34;
	int32	id35 = 35;
	int32	id36 = 36;
	int32	id37 = 37;
	int32	id38 = 38;
	int32	id39 = 39;
	int32	id40 = 40;
	int32	id41 = 41;
	int32	id42 = 42;
	int32	id43 = 43;
	int32	id44 = 44;
	int32	id45 = 45;
	int32	id46 = 46;
	int32	id47 = 47;
	int32	id48 = 48;
	int32	id49 = 49;
	int32	id50 = 50;
	int32	id51 = 51;
	int32	id52 = 52;
	int32	id53 = 53;
	int32	id54 = 54;
	int32	id55 = 55;
	int32	id56 = 56;
	int32	id57 = 57;
	int32	id58 = 58;
	int32	id59 = 59;
	int32	id60 = 60;
	int32	id61 = 61;
	int32	id62 = 62;
	int32	id63 = 63;
	int32	id64 = 64;
	int32	id65 = 65;
	int32	id66 = 66;
	int32	id67 = 67;
	int32	id68 = 68;
	int32	id69 = 69;
	int32	id70 = 70;
	int32	id71 = 71;
	int32	id72 = 72;
	int32	id73 = 73;
	int32	id74 = 74;
	int32	id75 = 75;
	int32	id76 = 76;
	int32	id77 = 77;
	int32	id78 = 78;
	int32	id79 = 79;
	int32	id80 = 80;
	int32	id81 = 81;
	int32	id82 = 82;
	int32	id83 = 83;
	int32	id84 = 84;
	int32	id85 = 85;
	int32	id86 = 86;
	int32	id87 = 87;
	int32	id88 = 88;
	int32	id89 = 89;
	int32	id90 = 90;
	int32	id91 = 91;
	int32	id92 = 92;
	int32	id93 = 93;
	int32	id94 = 94;
	int32	id95 = 95;
	int32	id96 = 96;
	int32	id97 = 97;
	int32	id98 = 98;
	int32	id99 = 99;
	int32	id100 = 100;
}

message evenBiggerVarintDummy {
	int32	id1 = 1;
	int32	id2 = 2;
	int32	id3 = 3;
	int32	id4 = 4;
	int32	id5 = 5;
	int32	id6 = 6;
	int32	id7 = 7;
	int32	id8 = 8;
	int32	id9 = 9;
	int32	id10 = 10;
	int32	id11 = 11;
	int32	id12 = 12;
	int32	id13 = 13;
	int32	id14 = 14;
	int32	id15 = 15;
	int32	id16 = 16;
	int32	id17 = 17;
	int32	id18 = 18;
	int32	id19 = 19;
	int32	id20 = 20;
	int32	id21 = 21;
	int32	id22 = 22;
	int32	id23 = 23;
	int32	id24 = 24;
	int32	id25 = 25;
	int32	id26 = 26;
	int32	id27 = 27;
	int32	id28 = 28;
	int32	id29 = 29;
	int32	id30 = 30;
	int32	id31 = 31;
	int32	id32 = 32;
	int32	id33 = 33;
	int32	id34 = 34;
	int32	id35 = 35;
	int32	id36 = 36;
	int32	id37 = 37;
	int32	id38 = 38;
	int32	id39 = 39;
	int32	id40 = 40;
	int32	id41 = 41;
	int32	id42 = 42;
	int32	id43 = 43;
	int32	id44 = 44;
	int32	id45 = 45;
	int32	id46 = 46;
	int32	id47 = 47;
	int32	id48 = 48;
	int32	id49 = 49;
	int32	id50 = 50;
	int32	id51 = 51;
	int32	id52 = 52;
	int32	id53 = 53;
	int32	id54 = 54;
	int32	id55 = 55;
	int32	id56 = 56;
	int32	id57 = 57;
	int32	id58 = 58;
	int32	id59 = 59;
	int32	id60 = 60;
	int32	id61 = 61;
	int32	id62 = 62;
	int32	id63 = 63;
	int32	id64 = 64;
	int32	id65 = 65;
	int32	id66 = 66;
	int32	id67 = 67;
	int32	id68 = 68;
	int32	id69 = 69;
	int32	id70 = 70;
	int32	id71 = 71;
	int32	id72 = 72;
	int32	id73 = 73;
	int32	id74 = 74;
	int32	id75 = 75;
	int32	id76 = 76;
	int32	id77 = 77;
	int32	id78 = 78;
	int32	id79 = 79;
	int32	id80 = 80;
	int32	id81 = 81;
	int32	id82 = 82;
	int32	id83 = 83;
	int32	id84 = 84;
	int32	id85 = 85;
	int32	id86 = 86;
	int32	id87 = 87;
	int32	id88 = 88;
	int32	id89 = 89;
	int32	id90 = 90;
	int32	id91 = 91;
	int32	id92 = 92;
	int32	id93 = 93;
	int32	id94 = 94;
	int32	id95 = 95;
	int32	id96 = 96;
	int32	id97 = 97;
	int32	id98 = 98;
	int32	id99 = 99;
	int32	id100 = 100;
	int32	id101 = 101;
	int32	id102 = 102;
	int32	id103 = 103;
	int32	id104 = 104;
	int32	id105 = 105;
	int32	id106 = 106;
	int32	id107 = 107;
	int32	id108 = 108;
	int32	id109 = 109;
	int32	id110 = 110;
	int32	id111 = 111;
	int32	id112 = 112;
	int32	id113 = 113;
	int32	id114 = 114;
	int32	id115 = 115;
	int32	id116 = 116;
	int32	id117 = 117;
	int32	id118 = 118;
	int32	id119 = 119;
	int32	id120 = 120;
	int32	id121 = 121;
	int32	id122 = 122;
	int32	id123 = 123;
	int32	id124 = 124;
	int32	id125 = 125;
	int32	id126 = 126;
	int32	id127 = 127;
	int32	id128 = 128;
	int32	id129 = 129;
	int32	id130 = 130;
	int32	id131 = 131;
	int32	id132 = 132;
	int32	id133 = 133;
	int32	id134 = 134;
	int32	id135 = 135;
	int32	id136 = 136;
	int32	id137 = 137;
	int32	id138 = 138;
	int32	id139 = 139;
	int32	id140 = 140;
	int32	id141 = 141;
	int32	id142 = 142;
	int32	id143 = 143;
	int32	id144 = 144;
	int32	id145 = 145;
	int32	id146 = 146;
	int32	id147 = 147;
	int32	id148 = 148;
	int32	id149 = 149;
	int32	id150 = 150;
	int32	id151 = 151;
	int32	id152 = 152;
	int32	id153 = 153;
	int32	id154 = 154;
	int32	id155 = 155;
	int32	id156 = 156;
	int32	id157 = 157;
	int32	id158 = 158;
	int32	id159 = 159;
	int32	id160 = 160;
	int32	id161 = 161;
	int32	id162 = 162;
	int32	id163 = 163;
	int32	id164 = 164;
	int32	id165 = 165;
	int32	id166 = 166;
	int32	id167 = 167;
	int32	id168 = 168;
	int32	id169 = 169;
	int32	id170 = 170;
	int32	id171 = 171;
	int32	id172 = 172;
	int32	id173 = 173;
	int32	id174 = 174;
	int32	id175 = 175;
	int32	id176 = 176;
	int32	id177 = 177;
	int32	id178 = 178;
	int32	id179 = 179;
	int32	id180 = 180;
	int32	id181 = 181;
	int32	id182 = 182;
	int32	id183 = 183;
	int32	id184 = 184;
	int32	id185 = 185;
	int32	id186 = 186;
	int32	id187 = 187;
	int32	id188 = 188;
	int32	id189 = 189;
	int32	id190 = 190;
	int32	id191 = 191;
	int32	id192 = 192;
	int32	id193 = 193;
	int32	id194 = 194;
	int32	id195 = 195;
	int32	id196 = 196;
	int32	id197 = 197;
	int32	id198 = 198;
	int32	id199 = 199;
	int32	id200 = 200;
}
